contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -4639,7 +4638,7 @@ exec_stmt_execsql(PLtsql_execstate *estate,
 			estate->schema_name = NULL;
 		if (estate->trigdata)
 			inside_trigger = true;
-		need_path_reset = reset_search_path(stmt, old_search_path, &reset_session_properties, inside_trigger);
+		need_path_reset = reset_search_path(stmt, &old_search_path, &reset_session_properties, inside_trigger);
```

Comment:
need_path_reset and old_search_path can be combined into one var. If returned value is not NULL, use it to reset search path